src/plugins/intel_cpu/src/nodes/eltwise.cpp

```diff
@@ -2317,6 +2317,42 @@ void Eltwise::initSupportedPrimitiveDescriptors() {
     }
     outputPrecision = filterPrecision(outputPrecision, forcedPrec);
     } else {
+#endif
+#if defined(OV_CPU_WITH_SHL)
+    const bool useShl = true;
```

Comment:
I think this `useShl` variable is used to avoid the following code that corresponds to the reference situation. So Im not sure about whether to delete this variable.